appshell/appshell_extensions_win.cpp

```diff
@@ -939,7 +1055,20 @@ int32 WriteFile(ExtensionString filename, std::string contents, ExtensionString
     if (INVALID_HANDLE_VALUE == hFile)
         return ConvertWinErrorCode(GetLastError(), false); 
```

Comment:
Same comment here as well, handle left unclosed. `CloseHandle` is not being called before returning.